dymos/visualization/linkage/report.py

```diff
@@ -237,3 +240,23 @@ def _convert_dicts_to_lists(tree_dict, show_all_vars):
                 tree_dict['children'].append(child)
 
         tree_dict.pop(CBN)
+
+
+def _run_linkage_report(prob):
+    """ Function invoked by the reports system """
+
+    # Find all Trajectory objects in the Problem. Usually, there's only one
+    for sysname, sysinfo in prob.model._subsystems_allprocs.items():
+        if isinstance(sysinfo.system, dm.Trajectory):
+            traj = sysinfo.system
+            # Only create a report for a trajectory with multiple phases
+            if len(traj._phases) > 1:
+                report_filename = f'{sysname}_{_default_linkage_report_filename}'
+                report_path = str(Path(prob.get_reports_dir()) / report_filename)
+                create_linkage_report(traj, report_path)
+
+
+def _linkage_report_register():
+    rptsys.register_report('linkage', _run_linkage_report, _default_linkage_report_title,
```

Comment:
Minor change, but let's make it obvious that the linkage report is part of dymos by registering it as `dymos.linkage`.